### PR TITLE
Rename whitelist

### PR DESCRIPTION
**Pull request by jdorweiler**
**Reviewer:**

## Description:
Copy changes to remove 'whitelist'.

## Steps to test this PR:
1. In the popup: "manage whitelist" -> "manage protection"
2. Hamburger menu: "manage whitelist and other settings" -> "manage protection and other settings"
3. On the settings page:
  - "Whitelisted Sites" -> "Unprotected Sites"
  - "No whitelisted sites" -> "Privacy Protection is on for all sites"
  - "Add site to whitelist" -> "Enter URL to disable protection"
4. When submitting broken site feedback the new message should be: "To fix the issue for the time being, you can turn off "Privacy Protection" for the site."

## Automated tests:
- [ ] Unit tests
- [ ] Integration tests

###### Reviewer Checklist:
- [ ] **Ensure the PR solves the problem**
- [ ] **Review every line of code**
- [ ] **Ensure the PR does no harm by testing the changes thoroughly**
- [ ] **Get help if you're uncomfortable with any of the above!**
- [ ] Determine if there are any quick wins that improve the implementation

###### PR Author Checklist:
- [ ] Get advice or leverage existing code
- [ ] Agree on technical approach with reviewer (if the changes are nuanced)
- [ ] Ensure that there is a testing strategy (and documented non-automated tests)
- [ ] Ensure there is a documented monitoring strategy (if necessary)
- [ ] Consider systems implications

### CODE REVIEW

File: shared/js/ui/templates/hamburger-menu.es6.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ module.exports = function () {
             <li>
                 <a href="javascript:void(0)" class="menu-title js-hamburger-menu-options-link">
                     Settings
-                    <span>Manage whitelist and other options</span>
+                    <span>Manage protection and other options</span>
```
Comment (alohaas) [inline shared/js/ui/templates/hamburger-menu.es6.js:17]: @jdorweiler should have caught this in the task, but "Protection" should be capitalized.

File: shared/js/ui/templates/whitelist.es6.js [hunk 2]
```diff
@@ -14,10 +14,10 @@ module.exports = function () {
     function addToWhitelist () {
         return bel`<div>
     <p class="whitelist-show-add js-whitelist-show-add">
-        <a href="javascript:void(0)">Add site to whitelist</a>
+        <a href="javascript:void(0)">Enter URL to disable protection</a>
```
Comment (alohaas) [inline shared/js/ui/templates/whitelist.es6.js:17]: @jdorweiler can we capitalize "Protection" here too?

**Comment by MariagraziaAlastra** [comment]
Looks good! Automated tests fail for reddit grading, but it works as expected when testing manually.
Merging, thanks!

**Comment by bgwines** [comment]
thanks so much for doing this!

**Pull request merged**